src/shared/components/download/__tests__/Download.spec.tsx

```diff
@@ -90,18 +95,39 @@ describe('Download component', () => {
     [FileFormat.tsv, true],
   ])(
     'should show column selection component when %s file type is selected and otherwise hide it',
-    (value, columnSelect) => {
+    async (value, columnSelect) => {
       const formatSelect = screen.getByTestId('file-format-select');
-      fireEvent.change(formatSelect, { target: { value } });
-      const customise = screen.queryByText('Customize data');
-      if (columnSelect) {
-        expect(customise).toBeInTheDocument();
-      } else {
-        expect(customise).not.toBeInTheDocument();
-      }
+      await act(async () => {
```

Comment:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#wrapping-things-in-act-unnecessarily